pypesto/objective/amici/amici.py

```diff
@@ -650,7 +650,9 @@ def check_gradients_match_finite_differences(
         """
         if x is None and "petab_problem" in dir(self.amici_object_builder):
             x = self.amici_object_builder.petab_problem.x_nominal_scaled
-            x_free = self.amici_object_builder.petab_problem.x_free_indices
+            kwargs = {
+                "x_free": self.amici_object_builder.petab_problem.x_free_indices
+            }
         return super().check_gradients_match_finite_differences(
-            *args, x=x, x_free=x_free, **kwargs
+            *args, x=x, **kwargs
```

Comment:
Is there a specific reason for those changes? I would feel like keeping an argument explicit always gives more info.

Reply:
This was necessary to fix the error described in #1494